MAUI/Calendar/selections.md

```diff
@@ -0,0 +1,249 @@
+* In this scenario, the `selection changed` event will return the first  and last date of the month, year, decade or century of the selected cell when the seletionmodeis set to `range `.
+Eg: 
+* In the year view, when the range is selected as Sep - Dec, then the range value will be 01-09-2022 to 31-12-2022.
+* In the decade view, when the range is selected as 2022 - 2025, then the range value will be 01-01-2022 to 31-12-2025.
+* In the century view, when the range is selected as 2020-2029 to 2030-2039, then the range value will be 01-01-2020 to 31-12-2039.
+
+### Range selection direction
+The Range Selection Direction is specified in the calendar property enumeration, which is used to select the range based on the direction. The `RangeSelectionDirection` provides five types of direction such as `Default`, `Forward`, `Backward`, `Both`, and `None`. The default [RangeSelectionDirection](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_RangeSelectionDirection) is `Default` in the SfCalendar.
+
+#### Range selection direction - forward
+If you set the `RangeSelectionDirection` property to `Forward`, you can select only the dates after the selected range start date, while dates before the range start date are considered disabled. The start date will not be changed.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        SelectionMode="Range"
+                        RangeSelectionDirection="Forward">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.SelectionMode = CalendarSelectionMode.Range;
+this.Calendar.RangeSelectionDirection = CalendarRangeSelectionDirection.Forward;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Forward range selection in .NET MAUI Calendar.](images/selection/maui-forward-range-selection.png)
+
+#### Range selection direction - backward
+If you set the `RangeSelectionDirection` property to `Backward`, you can select only the dates before the selected range end date, while dates after the range end date are considered disabled. The end date will not be changed.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        SelectionMode="Range"
+                        RangeSelectionDirection="Backward">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.SelectionMode = CalendarSelectionMode.Range;
+this.Calendar.RangeSelectionDirection = CalendarRangeSelectionDirection.Backward;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Backward range selection in .NET MAUI Calendar.](images/selection/maui-backward-range-selection.png)
+
+#### Range Selection Direction - Both
```

Comment:
Check Both - B is capital but other topics have small letters

Reply:
Both is changed to both